tests/util/verify.sh

```diff
@@ -237,7 +237,7 @@ __verify_with_retry() {
         local funcret="$?"
 
         # shellcheck disable=SC2001
-        out=$(sed 's/[[:space:]]*$//g' <<< "$out")
+        out=$(sed -e 's/[[:space:]]*$//g' -e '/./,$!d' <<< "$out")
```

Comment:
Deleting leading empty lines seems like it will cause confusion. If the output is expected to have empty lines at the start, we should include it in the expected output, or use `_verify_contains` if that's more convenient.

Deleting trailing spaces, on the other hand, is needed. It is solving the problem where sometimes the output has trailing space, but because of the lint checker does not allow trailing spaces in .md files, we can't include it in the expected output.

Reply:
@frankbu I'm fine moving to _verify_contains, but I found no way to actually add the empty lines at the start. If I add them in the index.md, it's carried to the snips, but the _verify_like(I think that's what that was using) still failed.

Reply:
Hmm, that sound like there may be a bug in the verify function. Handling white space properly has always been hard to get right because bash read does some stripping itself. If you can reproduce an example, open an issue for it.

Reply:
Removed and will fix the failing test case. Will add TODO: and comment here in case someone wants to see about a bug in the verify.

Reply:
Interestingly, the error with the empty line either fixed itself as it's not reappearing after the change.